otter/models/intents.py

```diff
@@ -1,29 +1,38 @@
 from functools import partial
 
+import attr
+
 from characteristic import attributes
 
-from effect import TypeDispatcher
+from effect import TypeDispatcher, sync_performer
 
 from twisted.internet.defer import inlineCallbacks, returnValue
 
 from txeffect import deferred_performer
 
+from otter.models.cass import CassScalingGroupServersCache
+from otter.util.fp import assoc_obj
+
 
-@attributes(['scaling_group', 'modifier'])
-class ModifyGroupState(object):
+@attr.s
+class ModifyGroupStateActive(object):
```

Comment:
I was wondering what the reason was for this change to `ModifyGroupState`?  I thought part of the point of putting the active boolean in the server cache was to eliminate writing to the state (and hence eliminate one more get a lock)?  Just modifying the active column also grabs a lock, so this doesn't seem to reduce lock contention.